packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -334,7 +334,8 @@ export function buildWebpackBrowser(
               const seen = new Set<string>();
               for (const file of emittedFiles) {
                 // Assets are not processed nor injected into the index
-                if (file.asset) {
+                // WorkerPlugin adds worker files to assets
+                if (file.asset && !file.file.endsWith('.worker.js')) {
```

Comment:
I am not following how this will fix this issue.

How is the reference the file change? When doing `new WebWorker`

Reply:
Or that’s part of the runtime js substitutions which we already handle?

Reply:
That makes it downlevel the file. Webpack doesn’t know that it exists outside of it being an asset.

Reply:
Implemented the second half of this that will update the file references.